src/transformers/__init__.py

```diff
@@ -115,8 +116,6 @@
         "is_tensorboard_available",
         "is_wandb_available",
     ],
-    "lib_integrations": [],
```

Comment:
Missing an `integrations` key here.

Reply:
That key is already defined above: https://github.com/younesbelkada/transformers/blob/move-integrations/src/transformers/__init__.py#L108